CHANGELOG.md

```diff
@@ -2,11 +2,29 @@
 
 ## HEAD
 
+## v.1.5.0
+
```

Comment:
I wonder if it's worth emphasising this one a bit?

How about something like this:
> 
> ### Storage
> 
> Ephemeral nodes are no-longer written for any tree by default (and have not been read since the `v1.4.0` release), the corresponding `---tree_ids_with_no_ephemeral_nodes` flag is now deprecated and will be removed in a future release).
>
?

Reply:
That is more clear, thanks!